cli/gardener_ci/cli_gen.py

```diff
@@ -116,8 +116,19 @@ def add_module(module_name, parser):
     try:
         module = __import__(module_name)
     except ImportError as ie:
-        if ie.name in ('containerregistry', 'kubernetes'):
-            return # XXX HACK: ignore this particular import error for now
+        if ie.name in (
+            'containerregistry',
+            'kubernetes',
+            'whitesource_common',
+            'sqlalchemy',
+            'psycopg2-binary',
+        ):
```

Comment:
also: is it not sufficient (/ more correct) to use the module names from cli/gardener_ci/*.py?